taskboot/utils.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def download_progress(url, path):
```

Comment:
Given that this is downloading from S3, it might be worth retrying in case of an intermittent failure

Reply:
(if you don't do it in this PR, please file an issue so we don't forget)

Reply:
:+1: